Assets/Editor/CreatureHydrator.cs

```diff
@@ -181,6 +185,37 @@ private static void HydrateCreature(UnitDefinition unit, string generaName, stri
         {
             spriteRenderer.sprite = idleSprite;
         }
+        
+        // Add a CapsuleCollider2D component
+        var collider = unitPrefab.AddComponent<CapsuleCollider2D>();
+        // Set the size of the collider
+        if (idleCanvasSize != null)
+        {
+            // Normalize the canvas size by 64 and then scale by 0.3 (this is what we had for the original dude)
+            float normalizedWidth = (float)idleCanvasSize.width / 64f;
+            float normalizedHeight = (float)idleCanvasSize.height / 64f;
+
+            collider.size = new Vector2(normalizedWidth * 0.3f, normalizedHeight * 0.3f);
+        }
+
+        collider.isTrigger = true;
+        collider.usedByEffector = false;
+        collider.direction = CapsuleDirection2D.Vertical;
+
+        // Add a Rigidbody2D component
+        var rigidbody = unitPrefab.AddComponent<Rigidbody2D>();
+        // Set Rigidbody2D properties
```

Comment:
are any of these the default or nah?

Reply:
no clue xD i just set everything that was set in the UI